src/components/header/index.module.scss

```diff
@@ -133,4 +82,50 @@
     border-right: 6px solid transparent;
     margin: 15px;
   }
-} 
\ No newline at end of file
+} 
+
+.imgContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 35px;
+  border-radius: 50%;
+  background-color: $white;
+}
+
+.selectContainer {
+  width: 270px;
+}
+
+select {
+  position: relative;
+  display: inline-block;
+  border: none;
+  margin-left: 10px;
+  padding: 0 34px 0 5px;
+  color: $white;
+  font-family: spurs;
+  font-size: 22px;
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  cursor: pointer;
+  appearance: none;
+  background: url('../../assets/icons/dropdownWhite.svg');
+  background-repeat: no-repeat;
+  background-position-x: 97%;
+  background-position-y: 50%;
```

Comment:
Can you explain why you decided to use 97%, not that it's a big deal, just wondering why, that's all.

Reply:
Sure. I couldnt render the select field width dynamically, based on team name. So I took the longest team name, minnesota timberwolves, and tried to get the right padding of the dropdown arrow, based on the design. That was 97%. Maybe in a later issue I can switch to react-select and enhance this section.